src/server/db/models/github-user.js

```diff
@@ -0,0 +1,6 @@
+import db from '../index';
+
+export const GitHubUser = db.Model.extend({
+  tableName: 'github_user',
```

Comment:
Is the schema of the table only defined in the migration file?

It would be useful to be able to find available fields in model file.